src/live.asp.net/TagHelpers/ClickToShowTagHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Mvc.TagHelpers;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace live.asp.net.TagHelpers
+{
+    [TargetElement("clicktoshow", Attributes = "value")]
+    public class ClickToShowTagHelper : TagHelper
+    {
+        public string Value { get; set; }
+
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = "p";
+
+            var p = new TagBuilder("p");
+            p.AddCssClass("click-to-show");
```

Comment:
I left out the `form-control-static` class from the taghelper to not bind it to bootstrap but just to the **click-to-show** behavior. 
On the other side, since this taghelper is always used for static text in a `p` tag and the website is using bootstrap, so maybe we could save some byte putting the `form-control-static` class in the taghelper.

@DamianEdwards ?